src/digitalIdentity-frontend/src/app/pages/login-page/login-page.component.html

```diff
@@ -40,6 +40,7 @@
           <mat-icon>{{ hide ? "visibility_off" : "visibility" }}</mat-icon>
         </mat-icon>
       </mat-form-field>
+      <a href="password/change"> Change password?</a>
```

Comment:
To access the password change page I have to click on forgotten password. If I have forgotten my password how can I change it if I don't know the old one? because we need the old one to create the new one. the name should be changed to better describe what the password change page does

Reply:
Yes you are right. I did not implement forgottten password. It is in our backlog: #131